user_modules/script_runner.lua

```diff
@@ -0,0 +1,134 @@
+local Test = require('user_modules/dummy_connecttest')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+local stepNumber = 0
+local runnerState = {
+  precondition = true,
+  test = true,
+  postcondition = true,
+  title = false
+}
+local title
+local steps = {}
+
+local runner = {}
+
+
+
+--[ATF]
+local function buildStepName(testStepName)
+  if type(testStepName) == "string"
+    and testStepName ~= "" then
+    testStepName = testStepName:gsub("%W", "_")
+    while testStepName:match("^[%d_]") do
+      if #testStepName == 1 then
+        stepNumber = stepNumber + 1
+        return "Unknown_test_step_" .. stepNumber
+      end
+      testStepName = testStepName:sub(2)
+    end
+    if testStepName:match("^%l") then
+        testStepName = testStepName:sub(1, 1):upper() .. testStepName:sub(2)
+    end
+    return testStepName
+  elseif type(testStepName) == "number" then
+    return "Test_step_" .. testStepName
+  else
+    stepNumber = stepNumber + 1
+    return "Unknown_test_step_" .. stepNumber
+  end
+end
+
+local function buildStepImplFunction(testStepImplFunction)
+  if type(testStepImplFunction) == "function" then
+    return testStepImplFunction
+  else
+    return function()
```

Comment:
@LuxoftAKutsan Default test step function generation was replaced with code that stop script execution with descriptive comment.